modules/landing_zone/main.tf

```
@@ -65,13 +65,14 @@ resource "null_resource" "landing_zone_destroy" {
     when    = "create"
     command = "echo 'info: apply ignored because part of destroy'"
   }
-  
+
   "provisioner" "local-exec" {
```

Argument names should not be quoted
```suggestion
   provisioner "local-exec" {
```